api/v1alpha1/workflow_types.go

```diff
@@ -137,7 +143,7 @@ type WorkflowDriverStatus struct {
 type WorkflowStatus struct {
 	// The state the resource is currently transitioning to.
 	// Updated by the controller once started.
-	// +kubebuilder:default=proposal
+	// +kubebuilder:default:=proposal
```

Comment:
In the old line, would this just not use the default then?

Reply:
Looks like "make manifests" accepts either.  The docs show :=